### PR TITLE
Fixed issue #204: Added tooltip for the buttons to explain what they do

### PR DESCRIPTION
**Pull request by Devyani-Divs**
Added tooltip for "Bugs and builds" button, in order to explain that they get bugs and builds about the package. @ralphbean @pypingou

### CODE REVIEW

File: pkgdb2/templates/package.html [hunk 1]
```diff
@@ -72,7 +72,7 @@
                     if package.monitoring_status == False %} class="active"{% endif
                     -%}>No monitoring</label> <br />
             <input type="radio" id="monitoring" name="monitoring" value="1" />
-                <label for="monitoring" {%-
+                <label for="monitoring" title="Lists bugs and builds for the package" {%-
```
Comment (pypingou) [inline pkgdb2/templates/package.html]: Good idea but the text is wrong

**Comment by Devyani-Divs** [comment]
@pypingou edited the text, does it look ok?

**Comment by pypingou** [comment]
@Devyani-Divs it does not reflect what these buttons do, try to find the button in the UI, and check http://fedoraproject.org/wiki/Upstream_release_monitoring

**Comment by ralphbean** [comment]
@Devyani-Divs, 01b490c comes closer, but there are three buttons there and this includes a tooltip for only one of them.  For completeness' sake, there should probably be a tooltip for each of the three explaining what each one does.

**Comment by Devyani-Divs** [comment]
@ralphbean apologies ! Doing it right away !

File: pkgdb2/templates/package.html [hunk 2]
```diff
@@ -68,15 +68,15 @@
         <td colspan="2" align="center">
           <div id="monitoring_block">
             <input type="radio" id="no_monitoring" name="monitoring" value="0" />
-                <label for="no_monitoring" {%-
+                <label for="no_monitoring" title="Will toggle the monitor flag to monitor package for new releases" {%-
```
Comment (pypingou) [inline pkgdb2/templates/package.html]: What about something more direct, like: `Turns off monitoring new releases for this package`

File: pkgdb2/templates/package.html [hunk 3]
```diff
@@ -68,15 +68,15 @@
...
                     if package.monitoring_status == False %} class="active"{% endif
                     -%}>No monitoring</label> <br />
             <input type="radio" id="monitoring" name="monitoring" value="1" />
-                <label for="monitoring" {%-
+                <label for="monitoring" title="Will toggle the monitor flag to monitor bugs and the buildsystem for new builds" {%-
```
Comment (pypingou) [inline pkgdb2/templates/package.html]: Here maybe we could have something like: `Turns on monitoring and attempt to build new releases for this package`

File: pkgdb2/templates/package.html [hunk 4]
```diff
@@ -68,15 +68,15 @@
...
                     if package.monitoring_status == True %} class="active"{% endif
                     -%}>Bugs &amp; builds</label> <br />
             <input type="radio" id="no_build" name="monitoring" value="nobuild" />
-                <label for="no_build" {%-
+                <label for="no_build" title="Will toggle the monitor flag to monitor bugs for the package" {%-
```
Comment (pypingou) [inline pkgdb2/templates/package.html]: And finally: `Turns on monitoring of new releases but without attempt to build it`

What do you think?

Comment (Devyani-Divs) [inline pkgdb2/templates/package.html]: @pypingou what can i say, apart from "just perfect" :)

**Comment by pypingou** [comment]
Fixed by https://github.com/fedora-infra/pkgdb2/pull/247

**Pull request closed without merging**